roles/ansible-keylime-tpm20/meta/main.yml

```diff
@@ -0,0 +1,57 @@
+galaxy_info:
+  author: Luke Hinds
+  description: Ansible Keylime
+  company: Red Hat
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (Apache License 2.0)
+
+  min_ansible_version: 1.2
+
+  # If this a Container Enabled role, provide the minimum Ansible Container version.
+  # min_ansible_container_version:
+
+  # Optionally specify the branch Galaxy will use when accessing the GitHub
+  # repo for this role. During role install, if no tags are available,
+  # Galaxy will use this branch. During import Galaxy will access files on
+  # this branch. If Travis integration is configured, only notifications for this
+  # branch will be accepted. Otherwise, in all cases, the repo's default branch
+  # (usually master) will be used.
+  #github_branch:
+
+  #
+  # platforms is a list of platforms, and each platform has a name and a list of versions.
+  #
+  platforms:
+    - name: Fedora
+      versions:
+        - 28
```

Comment:
Any reason for not using 29, is it causing issues?

Reply:
In fact your changes are running against 29 in travis and passing.

Reply:
Because I thought Fedora 29 is using python3 instead of python2, which is needed by python-keylime, I haven't tried installation of python-keylime on Fedora 29 yet. That is why I am using Fed 28.